client/concordclient/include/client/concordclient/concord_client.hpp

```diff
@@ -187,7 +187,7 @@ class ConcordClient {
 
   logging::Logger logger_;
   const ConcordClientConfig& config_;
-  std::shared_ptr<concordMetrics::Aggregator> metrics_;
+  std::shared_ptr<concordMetrics::Aggregator> aggregator_;
```

Comment:
Nit: Would metrics_aggregator_ be very specific name?

Reply:
10-20% of aggregator in our code are called <some_name>_aggregator.
but mostly we just used the name aggregator_.
It's more like a convention in our code.
Anyway, the name metrics_ was a total mistake.
We can use the name concord_client_aggregator_
but I don't see how it helps.